backend/contentBackend.js

```diff
@@ -210,7 +208,7 @@ async function getAllContent() {
  * @return { string | undefined } - previously set homeRoute
  */
 async function getHomeRoute() {
-  return (await get('homeRoute') || undefined);
+  return (await get('homeRoute')) || undefined;
```

Comment:
I don't understand.  How is this different from saying just
```suggestion
  return await get('homeRoute');
```
?